osutil/settime_32bit.go

```diff
@@ -0,0 +1,39 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+// +build 386 arm
```

Comment:
I don't actually know if there are other architectures that we build snapd on that are 32-bit, happy to update this or have others update this if we know of others